src/app/routes/onDemandRadio/getInitialData/index.js

```diff
@@ -8,22 +8,13 @@ import pathWithLogging, {
 } from '../../../lib/utilities/logging/pathWithLogging';
 import { RADIO_MISSING_FIELD } from '#lib/logger.const';
 
-const getEpisodeAvailability = ({ availableFrom, availableUntil }) => {
-  const timeNow = Date.now();
-  if (!availableUntil || timeNow < availableFrom) {
+const getEpisodeAvailability = ({ availableUntil }) => {
+  if (!availableUntil) {
     return false;
   }
   return true;
 };
```

Comment:
This function now just returns the value of `availableUntil`, so it can be this:
```suggestion
const getEpisodeAvailability = ({ availableUntil }) => availableUntil;
```
But I'd recommend just deleting it 🙂

Reply:
Aye I considered this. `availableUntil` is the ms time value, so we do need some sort of parsing of it into a boolean. Agreed this implementation is a bit clunky! What do you think would be the most succinct way of doing that?

P.s. clunky but... _scalable_

Reply:
Aaah right - updated suggestion with `Boolean(availableUntil)`  😄

Reply:
Nice. Locked this option in